src/common/services/web3.js

```diff
@@ -1,5 +1,12 @@
 import Web3 from 'web3';
 
-const web3 = new Web3();
+const rpcUrl = process.env.RPC_URL || 'http://localhost:8545';
 
-export default web3;
+const provider = new Web3.providers.WebsocketProvider(rpcUrl);
+const web3 = new Web3(provider);
+
+const connect = async () => {
+  return (await web3.eth.getBlock('latest')).number;
+};
+
+export default { connect, web3, provider };
```

Comment:
You can add the export statements directly where the objects are defined, like:

`export default const web3 = ...`
`export function connect = ...`

The provider one can always retreive via https://web3js.readthedocs.io/en/v1.2.6/web3-bzz.html#currentprovider

Also I would maybe rename `connect` as the method is not "really" connecting to the blockchain (I understand `connect` as an required step to initialize a whole system while here it is used to just check if there IS a connection).